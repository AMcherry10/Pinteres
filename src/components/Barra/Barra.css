#iconoPinterest{
  color: brown;
  font-size: 30px;
  margin-top: 10px;
  margin-left: -80px;
}

#buscar{
  margin-top: 20px;
  margin-left: 15px;
}

#inputSearch{
  background-color: rgb(238, 236, 236);
  border-radius: 5px;
  margin-left: 0px;
  margin-top: 10px;
  border: rgb(238, 236, 236);  
}

#inputSearch:focus{
  outline: 0;
  box-shadow: 0 0 0 0.2rem transparent !important;
}

#opciones{
  background-color: rgb(238, 236, 236);
  border-radius: 5px;
  margin-top: 10px;
  border: rgb(238, 236, 236);
  margin-left: -15px;
  font-size: 12px;
  font-family: Arial, Helvetica, sans-serif;
  font-weight: bold;
  text-align: center;
}

#opciones:focus{
  outline: 0;
  box-shadow: 0 0 0 0.2rem transparent !important;
}

#iconoUser{  
  font-size: 20px;
  color: #a3a0a0;   
  float: right;
  margin-left: 20px;
  margin-top: 15px;
}

#barras{  
  color: rgb(146, 143, 143);
  font-size: 25px;  
  float: right;
  margin-left: 25px;
  margin-top: 15px;
}

#msn{  
  color: rgb(163, 160, 160);
  font-size: 25px; 
  float: right;
  margin-left: 23px;
  margin-top: 15px;
}

@media (max-width: 700px) {
  .icono{
    display: none;
  }
  #iconoPinterest{
    margin-left: 0px;
  }
  #barra{
    display: none;
  }
}

@media (max-width: 550px) {
  #inputSearch{
    margin-left: 25px;
  }
}

@media (max-width: 1200px) {
  #msn{
    display: none;
  }
}

@media (max-width: 800px) {
  #barras{
    display: none;
  }
}
