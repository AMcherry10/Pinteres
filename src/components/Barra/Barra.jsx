import React, { Component } from "react";
import "./Barra.css"

class Barra extends Component {
  render() {
    return (
      <div className="container">
        <div className="row">
          <div className="col-12 col-xs-12 col-md-12 col-lg-12 col-xl-12">
          <div className="row">
            <i id="iconoPinterest" className="fab fa-pinterest col-1"></i>
            <input id="inputSearch" className="col-8" type="text" placeholder="Buscar" />
            <select id="opciones" className="col-2">
              <option value="Todos los pines">Todos los pines</option>
            </select>
            <i id="iconoUser" className="fas fa-user-circle col-">Noelia</i>
            <i id="barras" className="fas fa-bars"></i>
            <i id="msn" className="fas fa-comment-dots"></i>
            </div>
          </div>
        </div>
      </div>
    )
  }
}

export default Barra;