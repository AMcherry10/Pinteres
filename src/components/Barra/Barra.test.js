import React from "react";
import Enzyme, { shallow, mount, render } from "enzyme";
import renderer from "react-test-renderer";
import Barra from "../Barra/Barra";
import Adapter from "enzyme-adapter-react-16"

Enzyme.configure({ adapter: new Adapter() });

describe("Barra Component", () => {
 
  // make our assertion and what we expect to happen 
  it("hace render de componente", () => {
    expect(shallow(<Barra />).find("div.container").exists()).toBe(true)
  })

  it("render correcto", () => {
    const component = renderer.create(
      <Barra/>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
 })