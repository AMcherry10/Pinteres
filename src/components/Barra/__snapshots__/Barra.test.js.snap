// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Barra Component render correcto 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-12 col-xs-12 col-md-12 col-lg-12 col-xl-12"
    >
      <div
        className="row"
      >
        <i
          className="fab fa-pinterest col-1"
          id="iconoPinterest"
        />
        <input
          className="col-8"
          id="inputSearch"
          onKeyPress={[Function]}
          placeholder="Buscar"
          type="text"
        />
        <select
          className="col-2"
          id="opciones"
        >
          <option
            value="Todos los pines"
          >
            Todos los pines
          </option>
        </select>
        <i
          className="fas fa-user-circle col-"
          id="iconoUser"
        >
          Noelia
        </i>
        <i
          className="fas fa-bars"
          id="barras"
        />
        <i
          className="fas fa-comment-dots"
          id="msn"
        />
      </div>
    </div>
  </div>
</div>
`;
